control/controldisplay/csv.go

```diff
@@ -0,0 +1,73 @@
+package controldisplay
+
+import (
+	"log"
+
+	"github.com/turbot/go-kit/helpers"
+	typehelpers "github.com/turbot/go-kit/types"
+	"github.com/turbot/steampipe/control/execute"
+)
+
+type GroupCsvRenderer struct {
```

Comment:
I think we can just call it `CSVControlRenderer` - group is a bit generic (I know it renders resultgroups, but I think we can be more general in the name)

Reply:
capitalisation should match CSVFormatter